src/content_sao.cpp

```diff
@@ -202,6 +202,28 @@ void UnitSAO::getAttachment(int *parent_id, std::string *bone, v3f *position,
 	*rotation = m_attachment_rotation;
 }
 
+void UnitSAO::clearAttachments(bool detach_childs)
```

Comment:
this function sounds like it sound clear children, rather than the parent too.
I'd prefer two functions - `detachFromParent` and `clearChildAttachments` or `clearChildren`

Reply:
How about `clearParentAttachment` and `clearChildAttachments` to keep the function naming more or less consistent?